src/main/java/com/powsybl/openloadflow/network/LoadFlowType.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openloadflow.network;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public enum LoadFlowType {
+    AC,
+    DC
+}
```

Comment:
Maybe add a small comment so that newcomers may understand AC / DC difference? Something like 
```java
public enum LoadFlowMethod {
    /** Conventional non-linear loadflow method */
    AC,
    /** Linear MW-only loadflow method */
    DC
}
```

Reply:
Well forget it, that's an internal enum in fact, it would not be very useful